sklearn/tree/_tree.pyx

```diff
@@ -699,9 +699,6 @@ cdef class Tree:
         The current capacity (i.e., size) of the arrays, which is at least as
         great as `node_count`.
 
-    max_depth : int
```

Comment:
The `max_depth` is still an accessible value in `tree_`.

Reply:
Ah good catch. Fixed.